recipes/meamfit/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "meamfit" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/AndyDuff123/meamfit/-/archive/9b1b22823bd5810460891808aafca56b9c91d86d/meamfit-9b1b22823bd5810460891808aafca56b9c91d86d.tar.gz
```

Comment:
Is there a tag or release that we can use here instead of the hash?

Reply:
Unfortunately, I did not find any tag or release on the gitlab repository https://gitlab.com/AndyDuff123/meamfit . I am going to open an issue up-stream .

Reply:
Do you have something that links that hash to the 2.0.0 version that we can use here to document this?

Reply:
I took the version number from the manual https://gitlab.com/AndyDuff123/meamfit/-/blob/master/MEAMfitUserGuide.pdf

Reply:
If you take that version as the software release, which is already a big assumption, the hash should be the same as the manual commit: `4f66fd6bd6dea1066336e7e10ad7c1db132511b9`, would that work for you? Or are you using a newer one? If you are using a newer one, the safest thing to do would be to release it in a dev channel, something like 2.0.1.dev0

Reply:
That is a good point, I updated the recipe to use the hash from the manual commit.